README.md

```diff
@@ -29,45 +29,74 @@ climate science.
 
 ## Quick start
 
-Install Earth2Studio:
+Running AI weather prediction can be done with just a few lines of code.
 
-```bash
-pip install earth2studio[dlwp]
-```
+- See the [install guide][e2studio_install_url] for detailed steps for each model.
```

Comment:
Maybe change to "for detailed installation steps, including model-specific installations."?

Reply:
Good call fixed